adage/analyze/migrations/0009_auto_20170503_1700.py

```diff
@@ -1,28 +1,17 @@
 # -*- coding: utf-8 -*-
+
+"""This migration file used to generate a default participation type and
+set all Participation records to the default type.  These operations
+have been moved to fabfile/adage_server.py now.
```

Comment:
:+1:

Reply:
That is a good point. I will remove the `create_participation_type_if_nonexistent` function definition completely.